index.html

```diff
@@ -1,10 +1,13 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>JSON Web Token</title>
```

Comment:
```suggestion
    <title>JSON Web Token</title>
```
Give there is already a JWT spec, we might want to rename the title of the spec to something like: "Securing Verifiable Credentials using JSON Web Token". I expect that might need to become the by-line (because of the size of the font for spec titles) and the title might be something like "Using JWTs for Verifiable Credentials".

Reply:
I will handle content changes in a subsequent PR, I want to keep the content as is for this one.